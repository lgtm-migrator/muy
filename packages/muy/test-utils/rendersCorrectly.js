import renderer from "react-test-renderer"

const rendersCorrectly = component => {
  const tree = renderer.create(component).toJSON()
  return expect(tree).toMatchSnapshot()
}

export default rendersCorrectly
