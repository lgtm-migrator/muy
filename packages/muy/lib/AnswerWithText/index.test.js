import React from "react"
import AnswerText from "."
import rendersCorrectly from "../../test-utils/rendersCorrectly"

describe("Answer Text component", () => {
  it("renders correctly", () => {
    rendersCorrectly(<AnswerText />)
  })
})
