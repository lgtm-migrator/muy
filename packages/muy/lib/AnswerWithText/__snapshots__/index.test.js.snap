// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Answer Text component renders correctly 1`] = `
<dd
  className="MuiBox-root MuiBox-root-2"
  itemProp="acceptedAnswer"
  itemScope={true}
  itemType="https://schema.org/Answer"
>
  <div
    className="MuiBox-root MuiBox-root-3"
    itemProp="text"
  />
</dd>
`;
